staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -232,6 +234,7 @@ func ValidateCustomResourceDefinitionUpdate(ctx context.Context, obj, oldObj *ap
 		preexistingExpressions:                   findPreexistingExpressions(&oldObj.Spec),
 		versionsWithUnchangedSchemas:             findVersionsWithUnchangedSchemas(obj, oldObj),
 		celEnvironmentSet:                        environment.MustBaseEnvSet(environment.DefaultCompatibilityVersion()),
+		allowInvalidCABundle:                     allowInvalidCABundle(oldObj.Spec.Conversion),
```

Comment:
in ValidateCustomResourceDefinition, set `allowInvalidCABundle: true` since the CRD is not yet established or being served

Reply:
Fixed

Reply:
The ratcheting case already allows an invalid CA bundle to be updated to remain an invalid CA bundle. This case is only targeting changing a valid CA bundle -> invalid CA bundle pre-serving. Is there a use case for supporting this?